packages/migrate/src/__tests__/DbPush.test.ts

```diff
@@ -250,15 +255,104 @@ describeIf(process.platform !== 'win32')('push', () => {
     const result = DbPush.new().parse([])
     await expect(result).rejects.toMatchInlineSnapshot(`
 
-                                                                              ⚠️ We found changes that cannot be executed:
+      ⚠️ We found changes that cannot be executed:
 
-                                                                                • Made the column \`fullname\` on table \`Blog\` required, but there are 1 existing NULL values.
+        • Made the column \`fullname\` on table \`Blog\` required, but there are 1 existing NULL values.
 
-                                                                              Use the --force-reset flag to drop the database before push like prisma db push --force-reset
-                                                                              All data will be lost.
-                                                                                      
-                                                                `)
+      Use the --force-reset flag to drop the database before push like prisma db push --force-reset
+      All data will be lost.
+              
+    `)
     expect(ctx.mocked['console.log'].mock.calls.join('\n')).toMatchInlineSnapshot(``)
     expect(ctx.mocked['console.error'].mock.calls.join('\n')).toMatchInlineSnapshot(``)
   })
 })
+
+describeIf(process.platform !== 'win32' && !process.env.TEST_SKIP_MONGODB)('push existing-db with mongodb', () => {
```

Comment:
Do we have an issue that tracks that these tests are excluded for Windows completely because of Emoji? We should find a better solution for that :D

Reply:
Here's a related issue: https://github.com/prisma/prisma/issues/13355